test/XRP/default-xrp-client-test.ts

```diff
@@ -274,6 +274,31 @@ describe('Default XRP Client', function (): void {
     })
   })
 
+  it('Send XRP Transaction - success with memo and BigInteger', async function () {
+    // GIVEN an XRPClient, a wallet, and a BigInteger denomonated amount.
+    const xrpClient = new DefaultXRPClient(fakeSucceedingNetworkClient)
+    const { wallet } = Wallet.generateRandomWallet()!
+    const destinationAddress = 'X76YZJgkFzdSLZQTa7UzVSs34tFgyV2P16S3bvC8AWpmwdH'
+    const amount = bigInt('10')
+    const memos = [iForgotToPickUpCarlMemo]
+
+    // WHEN the account makes a transaction.
+    const transactionHash = await xrpClient.send(
+      amount,
+      destinationAddress,
+      wallet,
+      memos,
+    )
+
+    // THEN the transaction hash exists and is the expected hash
+    const expectedTransactionHash = Utils.toHex(
```

Comment:
I'd throw the expected vars under `GIVEN`, unless you have strong opinions about it being here. 

That way the test reads as:
GIVEN x
and an expected output from X

WHEN foo

THEN bar